deploy/docker/certificator/requirements.txt

```diff
@@ -1 +1,3 @@
--r ../base/requirements.txt
\ No newline at end of file
+-r ../base/requirements.txt
+jaeger_client
```

Comment:
Same that in https://github.com/eth-cscs/firecrest/pull/110/files#r667767457

Reply:
Fixed on 0c5b1fa